pynamodb/indexes.py

```diff
@@ -189,7 +190,16 @@ class GlobalSecondaryIndex(Index[_M]):
     """
     A global secondary index
     """
-    pass
+
+    @classmethod
+    def _get_schema(cls) -> Dict:
+        schema = super()._get_schema()
+        if getattr(cls.Meta, 'billing_mode', None) != PAY_PER_REQUEST_BILLING_MODE:
+            schema['provisioned_throughput'] = {
+                READ_CAPACITY_UNITS: cls.Meta.read_capacity_units,
+                WRITE_CAPACITY_UNITS: cls.Meta.write_capacity_units
+            }
+        return schema
```

Comment:
Here we add the GSI specific "ProvisionedThroughput" setting:
https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_CreateTable.html#DDB-CreateTable-request-GlobalSecondaryIndexes